dockerfiles/evmapp/entrypoint.sh

```diff
@@ -221,6 +222,11 @@ if [ "${SCNODE_FORGER_ENABLED:-}" = "true" ] || [ "${SCNODE_CERT_SUBMITTER_ENABL
   fi
 fi
 
+if [ -n "${SC_NODE_FORGER_REWARD_ADDRESS:-}" ]; then
+    FORGER_REWARD_ADDRESS="$(echo -en "\n        forgerRewardAddress = \"${SC_NODE_FORGER_REWARD_ADDRESS}\"")"
+fi
+export FORGER_REWARD_ADDRESS
+
```

Comment:
I would suggest moving this block under ```if [ "${SCNODE_FORGER_ENABLED:-}" = "true" ]; then``` line #231 since it is only applicable if the node is forger

Reply:
done - and added a warning in case is defined without  SCNODE_FORGER_ENABLED
Also: fixed a typo: SC_NODE_FORGER_REWARD_ADDRESS => SCNODE_FORGER_REWARD_ADDRESS